indico/modules/events/tracks/client/js/index.js

```diff
@@ -8,6 +8,34 @@
 (function(global) {
   'use strict';
 
+  function updateSorting(item) {
```

Comment:
`item` is always `$('#track-list-container')` so I'd just use that directly 2 lines below and remove the arg